src/main/java/seedu/address/commons/core/AddressbookComponentManager.java

```diff
@@ -7,17 +7,17 @@
  *
  * Registers the class' event handlers in eventsCenter
  */
-public class AnakinComponentManager {
+public abstract class AddressbookComponentManager {
```

Comment:
`AnakinComponentManager` was a duplicate of `ComponentManager`. We should delete it and also change `AnakinModelManager` (which is also the only place it's used) to extend `ComponentManager` instead, like how `LogicManager`, `UIManager`, and `StorageManager` all do

Reply:
I didn't rename this class directly, might be a side-effect of renaming another class. Resolved